vsctl/command/eventbus.go

```diff
@@ -59,40 +58,42 @@ func createEventbusCommand() *cobra.Command {
 			if eventbus == "" {
 				cmdFailedf(cmd, "the --name flag MUST be set")
 			}
-			_, err := client.CreateEventbus(context.Background(), &ctrlpb.CreateEventbusRequest{
+			eb, err := client.CreateEventbus(context.Background(), &ctrlpb.CreateEventbusRequest{
 				Name:        eventbus,
 				LogNumber:   eventlogNum,
 				Description: description,
-				NamespaceId: mustGetNamespaceID(primitive.DefaultNamespace).Uint64(),
+				NamespaceId: mustGetNamespaceID(namespace).Uint64(),
 			})
 			if err != nil {
 				cmdFailedf(cmd, "create eventbus failed: %s", Error(err))
 			}
 			if IsFormatJSON(cmd) {
-				data, _ := json.Marshal(map[string]interface{}{"Result": "Create Success", "EventbusService": eventbus})
+				data, _ := json.Marshal(map[string]interface{}{"Result": "Create Success", "ID": formatID(eb.Id), "Name": eventbus})
 				color.Green(string(data))
 			} else {
 				t := table.NewWriter()
-				t.AppendHeader(table.Row{"Result", "EventbusService"})
-				t.AppendRow(table.Row{"Success to create eventbus into [ default ] namespace", eventbus})
+				t.AppendHeader(table.Row{"Result", "ID", "Name"})
+				t.AppendRow(table.Row{"Success to create eventbus into [ default ] namespace", formatID(eb.Id), eventbus})
 				t.SetColumnConfigs([]table.ColumnConfig{
 					{Number: 1, VAlign: text.VAlignMiddle, Align: text.AlignCenter, AlignHeader: text.AlignCenter},
 					{Number: 2, AlignHeader: text.AlignCenter},
+					{Number: 3, AlignHeader: text.AlignCenter},
 				})
 				t.SetOutputMirror(os.Stdout)
 				t.Render()
 			}
 		},
 	}
-	cmd.Flags().StringVar(&eventbus, "name", "", "eventbus name to deleting")
+	cmd.Flags().StringVar(&namespace, "namespace", "", "namespace name to create eventbus, default name is default")
```

Comment:
cmd.Flags().StringVar(&namespace, "namespace", "default", "namespace name to create eventbus")